packages/core/android/src/main/java/io/embrace/rnembracecore/EmbraceManagerModule.java

```diff
@@ -571,6 +578,15 @@ public void recordCompletedSpan(String name, Double startTimeMs, Double endTimeM
         }
     }
 
+    public boolean isNetworkSpanForwardingEnabled() {
+        InternalNetworkApiImpl networkApi = new InternalNetworkApiImpl();
```

Comment:
would be good to avoid creating this instance on each call, either could try: `EmbraceInternalApi.getInstance().internalInterface.isNetworkSpanForwardingEnabled()` or if that doesn't work saving the instance on the class

Reply:
I was able to move the instance outside of the method only. I'm not sure if the SDK exposes the EmbraceInternalApi, or at least I couldn't use it. We can discuss about it during standup